extensions/amp-bind/0.1/test/validator-amp-bind.html

```diff
@@ -20,7 +20,7 @@
 <html ⚡>
 <head>
   <meta charset="utf-8">
-  <title>amp-bind: Basic</title>
+  <title>amp-bind</title>
```

Comment:
Thanks. Let me pull that one back to google3, I hadn't seen it, and then sync back out. I'll revert this and start over.